docs/docs/qualification.md

```diff
@@ -0,0 +1,68 @@
+# Simulation League: Registration & Qualification
+
+## Overview
+
+The RCJ Soccer Simulation League will be an additional league played at the 2021 RCJ
+and will be held in June fully-virtual. In order to participate in the **June Tournament**,
+you need to successfully qualify in one of our **Qualifiation Tournaments** in May.
+The qualification will be held centrally by the RCJ Soccer OC.
+
+You will need to register twice:
+
+- once for the Qualification Tournament, and
+- again for the final June Tournament (if qualified)
+
+
+## Timeline
+
+In order to participate in the RCJ Soccer Simulation League 2021 you must pass the Qualification
+stage that will be held centrally by the RCJ Soccer OC in May.
+
+- Qualification **registration** deadline: &nbsp; May 17'th 2021  
+- Qualification **code submission** deadline: &nbsp; May 24'th 2021
+
+We will publish the qualification results on June 2nd. If qualified, you will need to register
+again for the 2021 RCJ Championships in June:
+
+- June **registration** deadline: &nbsp; June 7'th 2021
+- June **code submission** deadline: &nbsp; June 14'th 2021
+(yes you can send in new code!)
+ 
+**Note:** Registering for the Qualification Tournament will be free from charge.
+The registration and code submission forms for the Qualification Tournament will be
+linked here in a few days. Please come back here for registration late April.
+
+## Qualification Tournaments
+
+There will be 3 virtual Super-Regional Qualification Tournaments in May, one for each region:
+
+- Americas
+- African + Europe
+- Asia + Pacific
+
+After the registration, teams will be assigned to their Super-Regional automatically.
+Each Super-Regional will play 7 rounds of Swiss System. After those rounds,
```

Comment:
> play 7 rounds of Swiss System

Can we perhaps link to the definition of the Swiss System somewhere? Would the Wiki page be good enough? (https://en.wikipedia.org/wiki/Swiss-system_tournament)

Reply:
Yes, sure, will do that in a separate commit asap :-)